pkgs/cupertino_http/lib/src/cupertino_client.dart

```diff
@@ -279,6 +279,15 @@ class CupertinoClient extends BaseClient {
       throw ClientException('Redirect limit exceeded', request.url);
     }
 
+    final headers = response.allHeaderFields
+        .map((key, value) => MapEntry(key.toLowerCase(), value));
```

Comment:
[optional] Another place we _could_ use destructuring and collection literals, but it's not a clear win.

```suggestion
    final headers = {
      for (final MapEntry(:key, :value) in response.allHeaderFields.entries)
        key.toLowerCase(): value
    };
```

Reply:
I don't actually think that this is better.